conf/broker.conf

```diff
@@ -102,6 +102,12 @@ tlsAllowInsecureConnection=false
 # Using a value of 0, is disabling unackeMessage limit check and consumer can receive messages without any restriction 
 maxUnackedMessagesPerConsumer=50000
 
+# Max number of unacknowledged messages allowed per shared subscription. Broker will stop dispatching messages to
+# all consumers once, this limit reaches until consumer starts acknowledging messages back and unack count reaches
+# to maxUnackedMessagesPerDispatcher/2. Using a value of 0, is disabling unackedMessage-limit check and dispatcher
+# can dispatch messages without any restriction
+maxUnackedMessagesPerDispatcher=200000
```

Comment:
> So, per-broker limit may not help broker??

I am not saying of doing that as part of this patch.. Just that if number of topic with bad behavior increase it could still use a lot of memory. Maybe doing something like that, when the threshold is reached, only block topics that have a lot of pending messages and leave alone topics with normal behavior.